src/assets/blobs/install-k3s

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
```

Comment:
Mostly just because it's also a great place for other files. But I can rename it `scripts` for now (which was what I originally had), and eventually rename it `blobs` if we get more things.